### PR TITLE
Remove models in favor of typeorm entities, and rebaselines migrations

### PR DESCRIPTION
**Pull request by jlhogan**

### CODE REVIEW

File: src/entity/Report.ts [hunk 1]
```diff
@@ -1,61 +0,0 @@
-import {
```
Comment (JordanGuinn) [inline src/entity/Report.ts:1]: Are these being removed cuz we don't actually need these report entities after all?  Or at least at this stage?

File: src/entity/embeddedColumns/UpdateMetaData.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import { Column, ManyToOne } from 'typeorm';
+import { ManyToOne, UpdateDateColumn } from 'typeorm';
 import { User } from '../User';
 
 export class UpdateMetaData {
-  @Column()
+  @UpdateDateColumn()
```
Comment (JordanGuinn) [inline src/entity/embeddedColumns/UpdateMetaData.ts:5]: Wow, seems like TypeORM has everything covered 👏🏽

File: src/entity/Report.ts (hunk 1 above)
Comment (jlhogan) [reply src/entity/Report.ts:1]: yeah that was my thinking -- no need to make this already more complicated than it needs to be app even more more complicated than it needs to be. And we have it in version control when/if we want it

File: src/entity/embeddedColumns/UpdateMetaData.ts (hunk 2 above)
Comment (jlhogan) [reply src/entity/embeddedColumns/UpdateMetaData.ts:5]: :)

**Pull request merged**